bp_common/src/include/bp_common_aviary_pkg.vh

```diff
@@ -152,6 +152,22 @@ package bp_common_aviary_pkg;
                         ,bp_unicore_cfg_p
                         );
 
+  localparam bp_proc_param_s bp_unicore_l1_hetero_override_p =
```

Comment:
We also need a config for a wide line config e.g. 512 bit line on a 16kB cache

Reply:
Added a config l1_wide where there are 2 16KB caches with 512-bit block widths.